source/install/desktop.rst

```diff
@@ -0,0 +1,18 @@
+===================================
+Desktop Applications 
+===================================
+
+Mattermost desktop applications are available for Windows, Mac and Linux operation systems. 
+
+They support all the features of the web experience, plus: 
+
+- Auto-start in Mattermost when a user logs into their machine.
+- Connect to multiple Mattermost servers from a single interface, and switch with shortcut keys.
+- Windows: Add Mattermost to Start menu, taskbar and system tray.
+- Mac: Add Mattermsot to the applications Dock
+
+You can `download them directly <https://about.mattermost.com/downloads/>`_, or use one fo the below installation guides:
+
+- `Windows Desktop installation guide <https://github.com/mattermost/desktop/blob/master/docs/setup.md#windows>`_
```

Comment:
Um... chocolatey just came out, propose waiting until it's got a bit more usage and feedback,